lib/command/delete.rb

```diff
@@ -5,28 +5,54 @@ class Delete < Base
     NAME = "delete"
     OPTIONS = [
       app_option(required: true),
+      workload_option,
       skip_confirm_option
     ].freeze
-    DESCRIPTION = "Deletes the whole app (GVC with all workloads, all volumesets and all images)"
+    DESCRIPTION = "Deletes the whole app (GVC with all workloads, all volumesets and all images) or a specific workload"
     LONG_DESCRIPTION = <<~DESC
-      - Deletes the whole app (GVC with all workloads, all volumesets and all images)
+      - Deletes the whole app (GVC with all workloads, all volumesets and all images) or a specific workload
       - Will ask for explicit user confirmation
     DESC
+    EXAMPLES = <<~EX
+      ```sh
+      # Deletes the whole app (GVC with all workloads, all volumesets and all images).
+      cpl delete -a $APP_NAME
```

Comment:
Looks like Thor defaults to the same name as the option:

![example](https://github.com/shakacode/heroku-to-control-plane/assets/25509361/b3799f7e-af19-478a-bde8-e7b7096aa90b)

So it will look for a workload named "workload".

Reply:
Please validate the name exists and is not `workload`